lib/JITServer/JITServer.cpp

```diff
@@ -174,6 +163,7 @@ ServerInitializeThreadContext(
     }
     catch (Js::OutOfMemoryException)
     {
+        CloseHandle((HANDLE)threadContextData->processHandle);
```

Comment:
Only thing that can throw here is the allocation, which should only throw OOM. Any other exception would be a bug (and will trigger a failfast in the content process).